drivers/i2c/i2c-priv.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _I2C_PRIV_H_
+#define _I2C_PRIV_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <i2c.h>
+#include <dt-bindings/i2c/i2c.h>
+
+static inline u32_t _i2c_map_dt_bitrate(u32_t bitrate)
+{
+	switch (bitrate) {
+	case I2C_BITRATE_STANDARD:
+		return I2C_SPEED_STANDARD << 1;
```

Comment:
I think we should still add
```
#define I2C_SPEED_SHIFT 1
```
to include/i2c.h file and use it here rather than hard coding the value.